connector/src/main/java/org/geysermc/connector/network/translators/block/entity/EndGatewayBlockEntityTranslator.java

```diff
@@ -0,0 +1,56 @@
+package org.geysermc.connector.network.translators.block.entity;
+
+import com.github.steveice10.opennbt.tag.builtin.CompoundTag;
+import com.github.steveice10.opennbt.tag.builtin.LongTag;
+import com.nukkitx.nbt.CompoundTagBuilder;
+import com.nukkitx.nbt.tag.IntTag;
+import com.nukkitx.nbt.tag.Tag;
+
+import java.util.ArrayList;
+import java.util.LinkedHashMap;
+import java.util.List;
+
+public class EndGatewayBlockEntityTranslator extends BlockEntityTranslator {
+    @Override
+    public List<Tag<?>> translateTag(CompoundTag tag) {
+        List<Tag<?>> tags = new ArrayList<>();
+        tags.add(new IntTag("Age", (int) (long) tag.get("Age").getValue()));
+        // Java sometimes does not provide this tag, but Bedrock crashes if it doesn't exist
+        // Linked coordinates
+        List<IntTag> tagsList = new ArrayList<>();
+        tagsList.add(new IntTag("", getExitPortalCoordinate(tag, "X")));
+        tagsList.add(new IntTag("", getExitPortalCoordinate(tag, "Y")));
+        tagsList.add(new IntTag("", getExitPortalCoordinate(tag, "Z")));
```

Comment:
Is this correct? Shouldn't this be lowercase?

Reply:
Yes; The ExitPortal CompoundTag in Java has the axis letters capitalized.

Reply:
:microjang: